oauthlib/oauth2/rfc6749/tokens.py

```diff
@@ -303,3 +301,47 @@ def estimate_type(self, request):
             return 5
         else:
             return 0
+
+
+class JWTToken(TokenBase):
+    __slots__ = (
+        'request_validator', 'token_generator',
+        'refresh_token_generator', 'expires_in'
+    )
+
+    def __init__(self, request_validator=None, token_generator=None,
+                 expires_in=None, refresh_token_generator=None):
+        self.request_validator = request_validator
+        self.token_generator = token_generator or random_token_generator
+        self.refresh_token_generator = (
+            refresh_token_generator or self.token_generator
+        )
+        self.expires_in = expires_in or 3600
+
+    def create_token(self, request, refresh_token=False, save_token=False):
+        """Create a JWT Token, using requestvalidator method."""
+
+        if callable(self.expires_in):
+            expires_in = self.expires_in(request)
+        else:
+            expires_in = self.expires_in
+
+        request.expires_in = expires_in
+
+        return self.request_validator.get_id_token(None, None, request)
```

Comment:
Similar to my other comment: What if someone wants to hand out JWT access tokens that are different from the ID Tokens they generate?

Reply:
Will rename this call to `get_jwt_bearer_token` and let explicit that for OIDC this sould call `get_id_token`